test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -116,7 +116,7 @@
     ('aten::dequantize', datetime.date(2020, 4, 1)),
     ('aten::confirmed_by_owner', datetime.date(2020, 3, 17)),
     ('aten::owner', datetime.date(2020, 3, 27)),
-    ('aten::owner_name', datetime.date(2020, 3, 27)),
+    ('aten::owner_name', datetime.date(2020, 4, 15)),
```

Comment:
why is this needed? we are changing `aten::owner` instead of `aten::owner_name` right?

Reply:
Removing it, since no BC test failure is triggered.

I don't know in what case op schema change will be treated as a BC breakage, only if used by production model?